### PR TITLE
fix log print and some nit issue

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
- fix some nit issue
#### Which issue(s) this PR fixes:

Fixes #
https://github.com/kubernetes-sigs/scheduler-plugins/issues/667
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: pkg/noderesourcetopology/README.md [hunk 1]
```diff
@@ -262,22 +262,22 @@ zones:
          $ kubectl get noderesourcetopologies.topology.node.k8s.io
        ```
 
-    1. Alternatively, in case you are just interested in simply testing the scheduler plugin, use the manifest in the manifest directory to deploy the CRD and CRs as follows:
+    2. Alternatively, in case you are just interested in simply testing the scheduler plugin, use the manifest in the manifest directory to deploy the CRD and CRs as follows:
```
Comment (ffromani) [inline pkg/noderesourcetopology/README.md:265]: markdown renderer should automatically convert the list index without the need for humans to enforce the correct ordering: https://www.markdownguide.org/basic-syntax/#ordered-lists

File: pkg/trimaran/resourcestats.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ func GetResourceLimits(pod *v1.Pod) *framework.Resource {
 	})
 }
 
-// GetEffectiveResource: calculate effective resources of a pod (CPU and Memory)
+// GetEffectiveResource : calculate effective resources of a pod (CPU and Memory)
```
Comment (ffromani) [inline pkg/trimaran/resourcestats.go:124]: why the extra space?

**Review by ffromani — COMMENTED** [review]
/ok-to-test

I'm not a fan of commits which fixes so many different issues in one go - unnecessarily hard to describe and bisect. But I won't block it either, let's see what other reviewers think.
However it seems some of the proposed changes can be simplified, please check inline comments

**Comment by googs1025** [comment]
> /ok-to-test
> 
> I'm not a fan of commits which fixes so many different issues in one go - unnecessarily hard to describe and bisect. But I won't block it either, let's see what other reviewers think. However it seems some of the proposed changes can be simplified, please check inline comments

Yes, normally I wouldn't submit a pull request like that. I just thought that these changes were all trivial, so it might be more convenient to merge them into a single pull request.

File: pkg/noderesourcetopology/README.md (hunk 1 above)
Comment (googs1025) [reply pkg/noderesourcetopology/README.md]: got it. done

File: pkg/trimaran/resourcestats.go (hunk 2 above)
Comment (googs1025) [reply pkg/trimaran/resourcestats.go]: I noticed that the method annotations in this directory were all like that. I simply reverted the changes to maintain code cleanliness. I have already rolled back the changes.
https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/pkg/trimaran/resourcestats.go#L77
```go
// GetResourceRequested : calculate the resource requests of a pod (CPU and Memory)
func GetResourceRequested(pod *v1.Pod) *framework.Resource {
	return GetEffectiveResource(pod, func(container *v1.Container) v1.ResourceList {
		return container.Resources.Requests
	})
}

// GetResourceLimits : calculate the resource limits of a pod (CPU and Memory)
func GetResourceLimits(pod *v1.Pod) *framework.Resource {
	return GetEffectiveResource(pod, func(container *v1.Container) v1.ResourceList {
		return container.Resources.Limits
	})
}

```

**Comment by googs1025** [comment]
@ffromani /PTAL thanks a lot

**Comment by googs1025** [comment]
@Huang-Wei /PTAL thanks!

**Review by Huang-Wei — COMMENTED** [review]
I agree with @ffromani that PRs should be organized by similar themes: like typo fixes, klog standardization, code de-duplication, etc.

But I don't want to block the merge either... Just next time please separate the changes instead of submitting a spaghetti-like one.

/lgtm
/approve

**Pull request merged**